otx/core/data/adapter/base_dataset_adapter.py

```diff
@@ -98,40 +102,38 @@ def _import_dataset(
             DatumaroDataset: Datumaro Dataset
         """
         dataset = {}
-        if train_data_roots is None and test_data_roots is None:
-            raise ValueError("At least 1 data_root is needed to train/test.")
+        if data_root is None:
+            if train_ann_file is None and test_ann_file is None:
+                raise ValueError("At least 1 data_root is needed to train/test.")
 
-        # Construct dataset for training, validation, testing, unlabeled
-        if train_data_roots:
-            # Find self.data_type and task_type
-            data_type_candidates = self._detect_dataset_format(path=train_data_roots)
-            self.data_type = self._select_data_type(data_type_candidates)
+        # Find self.data_type and task_type
+        data_type_candidates = self._detect_dataset_format(path=data_root)
+        self.data_type = self._select_data_type(data_type_candidates)
 
-            datumaro_dataset = DatumaroDataset.import_from(train_data_roots, format=self.data_type)
+        # Construct dataset for training, validation, testing, unlabeled
+        if data_root:
+            datumaro_dataset = DatumaroDataset.import_from(data_root, format=self.data_type)
+            dataset[Subset.TRAINING] = datumaro_dataset.get_subset("train")
+            dataset[Subset.VALIDATION] = datumaro_dataset.get_subset("val")
+            dataset[Subset.TESTING] = datumaro_dataset.get_subset("test")
 
+        if train_ann_file:
             # Prepare subsets by using Datumaro dataset
-            for k, v in datumaro_dataset.subsets().items():
-                if "train" in k or "default" in k:
-                    dataset[Subset.TRAINING] = v
-                elif "val" in k:
-                    dataset[Subset.VALIDATION] = v
-            self.is_train_phase = True
+            dataset[Subset.TRAINING] = DatumaroDataset.import_from(
```

Comment:
This is the hardest part when we used the Datumaro as an data frontend. Datumaro can't import annotation file directly. 
So, this line will not work.

Reply:
So, if we want to use annotation file directly, we need to change the Datumaro's function or use previous CLI datasets.

Reply:
![Screenshot from 2023-02-09 08-11-53](https://user-images.githubusercontent.com/13793254/217754487-869322c6-e82a-41c7-8bef-0800b0e88228.png)
@sungmanc it works for me without changing anything :thinking:

Reply:
> ![Screenshot from 2023-02-09 08-11-53](https://user-images.githubusercontent.com/13793254/217754487-869322c6-e82a-41c7-8bef-0800b0e88228.png) @sungmanc it works for me without changing anything 🤔

Wow, I had never thought about this way. Then, we can add direct annotation easily

Reply:
@sungmanc I did some tracing in the Datumaro code and seems like it works, but there might be some edge cases I haven't considered yet. Do you know what needs to be changed for a multi-classification folder structure?

Reply:
> @sungmanc I did some tracing in the Datumaro code and seems like it works, but there might be some edge cases I haven't considered yet. Do you know what needs to be changed for a multi-classification folder structure?

I'll look into deeper after the OTXv1.0 ! For now, I don't have idea.